obspy/signal/trigger.py

```diff
@@ -336,8 +336,6 @@ def trigger_onset(charfct, thres1, thres2, max_len=9e99, max_len_delete=False):
     # determine the indices where charfct falls below off-threshold
     ind2_ = np.empty_like(ind2, dtype=bool)
     ind2_[:-1] = np.diff(ind2) > 1
-    # last occurence is missed by the diff, add it manually
-    ind2_[-1] = True
```

Comment:
Not sure if we should keep this..

Reply:
actually I think this does not have to do with this bug and needs to stay, looking over the code a second time